front-end/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM node:18 as build-step
+
+RUN mkdir -p /app
```

Comment:
`WORKDIR /app` will create the dir given it does not exist, therefore this is unnecessary